src/crx.js

```diff
@@ -15,6 +16,7 @@ const DEFAULTS = {
   codebase: null,
   path: null,
   src: "**",
+  version: 2,
```

Comment:
Should it be rather `3`, now that's a default for Chrome?

Reply:
Yes. I was working on this patch before CRXv3 became mandatory :).
Fixed.